weasel/cli/run.py

```diff
@@ -304,37 +305,38 @@ def get_fileinfo(project_dir: Path, paths: List[str]) -> List[Dict[str, Optional
     return data
 
 
-def _check_requirements(requirements: List[str]) -> Tuple[bool, bool]:
+def _check_requirements(requirements: Path) -> None:
     """Checks whether requirements are installed and free of version conflicts.
-    requirements (List[str]): List of requirements.
-    RETURNS (Tuple[bool, bool]): Whether (1) any packages couldn't be imported, (2) any packages with version conflicts
-        exist.
+    Prints a warning and the output of 'pip install --dry-run' if problems are
+    detected.
+
+    requirements (Path): Path to requirements.
     """
-    import pkg_resources
-
-    failed_pkgs_msgs: List[str] = []
-    conflicting_pkgs_msgs: List[str] = []
-
-    for req in requirements:
-        try:
-            pkg_resources.require(req)
-        except pkg_resources.DistributionNotFound as dnf:
-            failed_pkgs_msgs.append(dnf.report())
-        except pkg_resources.VersionConflict as vc:
-            conflicting_pkgs_msgs.append(vc.report())
-        except Exception:
+    try:
+        import pip
+
+        if hasattr(pip, "__version__") and Version(pip.__version__) < Version("22.2"):
+            msg.warn("Unable to check requirements, please upgrade to pip 22.2+")
```

Comment:
This will get noisy, because the warning will be displayed prominently every time the user executes a `run` command. We can hook up `skip_requirements_check` with a new flag though on the CLI command though, so the user can disable the warning?

Reply:
I think a new CLI flag is overkill. This is just for python 3.6, and if it affects someone, they can use the standard flag in `project.yml`?

Reply:
Or you could just leave it out entirely and not warn here. I don't think more machinery around this is a good idea. If people have python 3.7+ they should just upgrade pip.

Reply:
I'd personally be in favor not having the warning, or maybe only as an `info` and not as a warning, which feels more disruptive / loud.

Reply:
I think it would be okay to remove it.

Reply:
Hmm, I think having no message might end up being kind of confusing, so I put it back in as `info`.